modules/users/client/controllers/admin/adduser.client.controller.js

```diff
@@ -0,0 +1,92 @@
+(function () {
```

Comment:
no need for this controller because we already have this exact controller in /modules/users/client/controllers.

also, bad practice duplicating code

also, I made changes to this in the file you copied and I don't see them here which means you should update your develop and rebase this branch off of it